src/api/index.tsx

```diff
@@ -155,8 +154,8 @@ type EmptyRequestAndResponse = {
 
 export const callApi = async <
   T extends {
-    requestData: undefined | {};
-    successfulResponse: {};
+    requestData: undefined | Record<string, unknown>;
+    successfulResponse: unknown;
```

Comment:
I know you don't really want to spend time fighting with TS (as discussed in our last engineering meeting), but I'm just wondering, how is `unknown` any better than `any`? This isn't code interacting with a third party, it's our own code, so, theoretically we should be able to have a proper type for this.

Reply:
I don't think `unknown` is better than `any` but the compiler suggested `unknown` to me so 🤷